packages/ra-ui-materialui/src/auth/Logout.tsx

```diff
@@ -13,12 +13,15 @@ interface Props {
     redirectTo?: string;
 }
 
-const useStyles = makeStyles((theme: Theme) => ({
-    menuItem: {
-        color: theme.palette.text.secondary,
-    },
-    icon: { minWidth: theme.spacing(5) },
-}));
+const useStyles = makeStyles(
+    (theme: Theme) => ({
+        menuItem: {
+            color: theme.palette.text.secondary,
+        },
+        icon: { minWidth: theme.spacing(5) },
+    }),
+    { name: 'LogoutWithRef' }
```

Comment:
Why not Logout?

Reply:
I was going by the export default. But I'll change it to the filename instead.